packages/shared/components/SelectorInput.svelte

```diff
@@ -92,7 +92,12 @@
                         on:click={() => onClick(option)}
                         class="w-full flex flex-row flex-1 justify-between items-center px-2 py-3 rounded-lg hover:bg-blue-50 dark:hover:bg-gray-800 dark:hover:bg-opacity-20"
                     >
-                        <div class="flex flex-row gap-3 justify-start items-center flex-1">
+                        <div
+                            class="flex flex-row gap-3 justify-start items-center flex-1 overflow-hidden {option.value &&
+                            option.key
+                                ? 'max-w-[50%]'
+                                : ''}"
```

Comment:
it works for me too without the need of `max-w-[50%]`, do we need it? 
![image](https://github.com/iotaledger/firefly/assets/3624944/b2d98ef7-c0fc-4156-9547-0f69fe20cb26)